specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -606,12 +612,12 @@
                   }
                 },
                 "ipAddress": {
-                  "description": "The IP address type.",
+                  "description": "The IP address type of the container group.",
                   "$ref": "#/definitions/IpAddress"
                 },
                 "osType": {
                   "type": "string",
-                  "description": "The base level OS type required by the containers in the group.",
```

Comment:
What do you mean by "base level" - host-level? Does the user need to know that, can they just input the OS type of the image they bring?